packages/serverless-components/domain/utils.js

```diff
@@ -81,7 +81,11 @@ const getOutdatedDomains = (inputs, state) => {
  * - These don't need to be created and SHOULD NOT be modified.
  */
 const getDomainHostedZoneId = async (route53, domain, privateZone) => {
-  const hostedZonesRes = await route53.listHostedZonesByName().promise();
+  const params = {
+    DNSName: domain
```

Comment:
Does this also work if domain has `www` in it or other subdomain?

Reply:
@dphang - Hey Daniel!  Great question.

Route 53 shouldn't ever return a subdomain from this list, since the listHostedZonesByName should only return the full domain that was used when creating the zone.

![image](https://user-images.githubusercontent.com/35877543/96378513-01325b80-1152-11eb-8b9f-3bac8675cd47.png)

Here is how Route 53 describes the "domain name" field when creating a zone.